### DISCUSSION TITLE
Blue Connect Wallet Button

### DISCUSSION THREAD

**Discussion by ReturnerNaGithub** [rainbow-me/rainbowkit#1577] [General]
How can i change the color blue button into the other colors?

![image](https://github.com/rainbow-me/rainbowkit/assets/78732470/4aeb76de-1b6a-4755-9354-6cf1b3dc451b)

**Comment by ReturnerNaGithub** [comment]
import React from 'react';
import { ResponsiveValue } from '../../css/sprinkles.css';
import { ConnectButtonRenderer } from './ConnectButtonRenderer';
type AccountStatus = 'full' | 'avatar' | 'address';
type ChainStatus = 'full' | 'icon' | 'name' | 'none';
export interface ConnectButtonProps {
    accountStatus?: ResponsiveValue<AccountStatus>;
    showBalance?: ResponsiveValue<boolean>;
    chainStatus?: ResponsiveValue<ChainStatus>;
    label?: string;
}
export declare function ConnectButton({ accountStatus, chainStatus, label, showBalance, }: ConnectButtonProps): React.JSX.Element;
export declare namespace ConnectButton {
    var __defaultProps: {
        readonly accountStatus: "full";
        readonly chainStatus: {
            readonly largeScreen: "full";
            readonly smallScreen: "icon";
        };
        readonly label: "Connect Wallet";
        readonly showBalance: {
            readonly largeScreen: true;
            readonly smallScreen: false;
        };
    };
    var Custom: typeof ConnectButtonRenderer;
}

export {};

**Comment by ReturnerNaGithub** [comment]
import { ConnectButton } from "@rainbow-me/rainbowkit";
import styles from "../../styles/Navbar.module.css";
import { useEffect, useState } from "react";
export default function Navbar() {
  return (
    <nav className={styles.navbar}>
    <img className={styles.alchemy_logo} alt={styles.logo} src="/belologo.png"></img>
      
         <a
        href="/minersnation"
        aria-label="open"
      ><img className={styles.about_logo} alt={styles.logo} src="/about.png"></img>
      </a>

      <a
        href="/investments"
        aria-label="open"
      ><img className={styles.invest_logo} alt={styles.logo} src="/investments.png"></img>
      </a>

      <a
        href="/"
        aria-label="open"
      ><img className={styles.rigparts_logo} alt={styles.logo} src="/my rig parts.png"></img>
      </a>

      <a
        href="/news"
        aria-label="open"
      ><img className={styles.news_logo} alt={styles.logo} src="/news.png"></img>
      </a>
      <ConnectButton>
      </ConnectButton>
    </nav>

  );
}

**Comment by DanielSinclair** [comment]
You can use a [Custom ConnectButton](https://www.rainbowkit.com/docs/custom-connect-button) to have full control, or use a [Theme](https://www.rainbowkit.com/docs/theming) or [Custom Theme](https://www.rainbowkit.com/docs/custom-theme). You can reference the applicable theme keys below:
```
connectButtonBackground: '...',
connectButtonBackgroundError: '...',
connectButtonInnerBackground: '...',
connectButtonText: '...',
connectButtonTextError: '...',
```
